web_accounts_list.json

```diff
@@ -4591,6 +4591,61 @@
            "known_accounts" : ["john", "junech"],
            "category" : "dating",
            "valid" : true
+          },
+          {
+           "name" : "Tunisie-Foot",
+           "check_uri" : "https://www.tunisie-foot.com/forum/wcf/user/{account}",
+           "account_existence_code" : "200",
+           "account_existence_string" : "profile",
+           "account_missing_string" : "Page Non Trouvée - Tunisie-Foot",
+           "account_missing_code" : "404",
+           "known_accounts" : ["7701-manager", "28487-tunisia-gang-city"],
+           "category" : "social",
+           "valid" : true
+          },
+          {
+           "name" : "ICQ Chat",
+           "check_uri" : "https://icq.icqchat.co/members/{account}",

```

Comment:
For this site, you need to add a `/` to the end of the `check_uri` since i'm getting a 301 response code instead of the 200 that is expected. The site redirects to the same URL but with a `/` on the end.